schemas/com.snowplowanalytics.monitoring.batch/load_succeeded/jsonschema/3-0-0

```diff
@@ -0,0 +1,185 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a data load succeeding",
+  "self": {
+    "vendor": "com.snowplowanalytics.monitoring.batch",
+    "name": "load_succeeded",
+    "format": "jsonschema",
+    "version": "3-0-0"
+  },
+
+  "type": "object",
+  "properties": {
+    "shredding": {
+      "description": "Information about the batch taken from transformation step, isomorphic to shredding_complete schema",
+      "type": "object",
+      "properties": {
+        "base": {
+          "description": "Blob storage path to the root of the batch",
+          "type": "string",
+          "format": "uri",
+          "maxLength": 1024
+        },
+        "compression": {
+          "description": "File compression type",
+          "enum": ["GZIP", "NONE"]
+        },
+        "typesInfo": {
+          "description": "Info about schemas used in events and output formats in the respective batch",
+          "type": "object",
+          "oneOf": [
+            {
+              "properties": {
+                "transformation": {
+                  "description": "Type of the transformation",
+                  "enum": ["SHREDDED"]
+                },
+                "types": {
+                  "description": "Set of Iglu URIs and the format they were shredded into",
+                  "type": "array",
+                  "items": {
+                    "properties": {
+                      "schemaKey": {
+                        "description": "Iglu URI",
+                        "type": "string",
+                        "maxLength": 256
+                      },
+                      "format": {
+                        "description": "File format in which the entities were shredded to",
```

Comment:
File format into which the entities were shredded

or

File format which the entities were shredded into

(Pedants like the former; plain English speakers say the latter)

Reply:
Let's be pedant then 😄